templates/acs-deployment-master.yaml

```diff
@@ -370,8 +370,6 @@ Parameters:
       Description: The Admin password for Alfresco
       NoEcho: True
     RegistryCredentials:
-      AllowedPattern: ".+"
-      ConstraintDescription: The private registry credentials can not be empty
       Type: String
```

Comment:
I think you still need a default:
Something like:

ewoJImF1dGhzIjogewoJCSJodHRwczovL3ByaXZhdGVyZWdpc3RyeS5jb20iOiB7CgkJCSJhdXRoIjogImRHVnpkRUIwWlhOMExtTnZiVHB3WVhOemQyOXlaQW89IgoJCX0sCgl9LAoJIkh0dHBIZWFkZXJzIjogewoJCSJVc2VyLUFnZW50IjogIkRvY2tlci1DbGllbnQvMTguMDYuMS1jZSAoZGFyd2luKSIKCX0sCgkic3RhY2tPcmNoZXN0cmF0b3IiOiAia3ViZXJuZXRlcyIKfSUK

which points to a bogus private registry. This should work.